client/request.go

```diff
@@ -376,6 +454,33 @@ func (r *Request) FormData(key string) []string {
 	return res
 }
 
+// FormDatas method returns all form datas in request instance.
+// You can use maps.Collect() to collect all cookies into a map.
+//
+// The returned value is valid until the request object is released.
+// Any future calls to FormDatas method will return the modified value. Do not store references to returned value. Make copies instead.
+func (r *Request) FormDatas() iter.Seq2[string, []string] {
```

Comment:
We should rename this since `Datas` is not an english word. An option is `AllFormData()`